docs/manual/manual_ifci_control.rst

```diff
@@ -7,23 +7,86 @@
+
+Highlighted in green are Control Values. A control value is an unsigned 16 bit value that defines a throttle, an X, or a Y command. X and Y commands are only applicable when 
+using Vertiq's :ref:`pulsing firmware <pulsing_module_start_guide>`. Any type of command can occupy any control value, and the location it occupies in an IFCI packet is called its Control 
+Value Index (CVI). For example, suppose you want to control a generic quadcopter. You may place the throttle commands for motor 1 in CVI 0, commands for motor 
+2 in CVI 1, and so on. Where each motor's throttle command is fully configurable, however, and it would be equally valid to place motor 4's throttle command in CVI 0.
+
+In order to bridge the flight controller's commands with the commands accepted by a module, Vertiq modules have their own version of the expected CVIs. For example, 
+suppose your flight controller is configured to transmit 10 control values. The throttle command for the module being configured is the 5th command (CVI 4) in the IFCI 
+packet. In order to have the module listen to that command, you simply set its `Throttle CVI` to 4. The same principle applies to X and Y commands as well.
+
+.. note::
+    If any module CVI is set to 255, the module will not look in any received control values for a command.
+
+All module Control Value Index values can be found in the :ref:`iquart_flight_controller_interface`. You can also find the CVIs in the :ref:`Control Center's <control_center_start_guide>` general tab:
+
+.. image:: ../_static/manual_images/pulsing_propeller/throttle_cvi_control_center.png
+    :align: center
+
+Mapping Received Control Values to Motor Control Outputs
+=============================================================
+As mentioned above, all IFCI control values are represented by unsigned 16 bit numbers representing an integer range from [0, 65535]. How these values are mapped 
+into commands applied by your module is dependent on the control value type (throttle, X, or Y) as well as other module configurations.
+
+Mapping Throttle Control Values to Applied Spin Throttle
+-----------------------------------------------------------
+Before continuing, please familiarize yourself with the basics of your module's Modes, Maximums, and Direction found :ref:`here <throttle_mode_maximums_directions>`. 
+
+Throttle control values are subject to the same behaviors governing both Hobby and general IQUART throttle inputs. In fact, throttle control values are processed by the same 
+pipeline as the :ref:`IQUART raw value <throttle_iquart_ref>`.
+
+In order to calculate the raw value that your module will apply, simply divide the received throttle control value by 65535. That value is then mapped to applied throttle based on your modes and directions.
+
+Suppose your module is configured to spin two-dimensionally counterclockwise with a 2D flight controller, and is set to velocity mode with a maximum of 1000 rad/s. 
```

Comment:
You mention a 2D flight controller here, I'm not really sure what that is? Is this referring to the FC Mode parameter? If you call it a 2D flight controller, I think everyone will be confused, its not a generally used term. Maybe just say "with FC Mode set to 2D" and link to the appropriate docs on that.